### DISCUSSION TITLE
How to translate the admin dashboard to spanish?

### DISCUSSION THREAD

**Discussion by Pacocc97** [medusajs/medusa#9986] [Q&A]
Hello my people. I can´t find a way to translate the admin dashboard to spanish. Does medusa has any native option for this? Or how can I manually translate the UI? 
I really need this future. 
Thanks!

**Comment by jpcbdev** [comment]
I need that too. I have to do this manually replacing the node_modules/@medusa/medusajs/admin/ui files -_-
